server/src/auth-router.ts

```diff
@@ -114,6 +118,14 @@ router.get(
       }
       response.redirect('/profile/settings?success=' + success.toString());
     } else {
```

Comment:
We should probably put the `challenge_team`-specific logic under an `else if (challenge_team) {`, right?

Also, at the bottom of the `else if` you're adding, please call:

```
response.redirect(redirect || basePath + 'login-success?ovchall=1');
```

That will let us show a different front-end flow for newly-enrolled users.

Reply:
If backend redirect `ovchall=1`, frontend will pop up the invite modal.
I guess it should and will be three new consecutive on-boarding modals in the recent future.
Thus, normally, invitation aside, there will be no ovchall=1 in the redirection, and no on-boarding modals.